descriptor-api/src/main/java/com/synopsys/integration/alert/descriptor/api/model/ChannelKeys.java

```diff
@@ -42,6 +45,8 @@ private static void putKey(ChannelKey channelKey) {
         ChannelKeys.putKey(ChannelKeys.JIRA_SERVER);
         ChannelKeys.putKey(ChannelKeys.MS_TEAMS);
         ChannelKeys.putKey(ChannelKeys.SLACK);
+        //FIXME: these ChannelKeys are used for testing the NotificationReceiverV2. These must be removed before releasing 6.5.0.
+        ChannelKeys.putKey(ChannelKeys.SLACK_V2);
```

Comment:
What if we remove the `@Component` annotation from the old `NotificationReceiver`? We're not using it anymore right?